store/src/java/com/zimbra/cs/service/mail/SendMsg.java

```diff
@@ -132,7 +132,8 @@ public Element handle(Element request, Map<String, Object> context, QName respQn
                boolean noSaveToSent = request.getAttributeBool(MailConstants.A_NO_SAVE_TO_SENT, false);
                boolean fetchSavedMsg = request.getAttributeBool(MailConstants.A_FETCH_SAVED_MSG, false);
                boolean deliveryReport = false;
-               if (LC.delivery_report_enabled.booleanValue()) {
+               String deliveryStatusNotificationEnabled = authAcct.getAttr(Provisioning.A_zimbraFeatureDeliveryStatusNotificationEnabled);
+               if ("TRUE".equalsIgnoreCase(deliveryStatusNotificationEnabled)) {
```

Comment:
since there is no further use of `deliveryStatusNotificationEnabled` can remove it and condition to
` if ("TRUE".equalsIgnoreCase(authAcct.getAttr(Provisioning.A_zimbraFeatureDeliveryStatusNotificationEnabled)))`